tests/mocks/mock_tool/tool.py

```diff
@@ -52,6 +53,15 @@ def test_str_output(self, value: dict) -> str:
     def test_no_schema(self, value: dict) -> str:
         return "no schema"
 
+    @activity(
+        config={
+            "description": "test description",
+            "schema": lambda _self: Schema(_self.custom_schema, description="Test input"),
```

Comment:
Also, just curious, why can't this be just `self` without the leading underscore?

Reply:
It can be called whatever you'd like since it's just a positional arg. I kept it as `_self` for consistency with the `_self` that is rendered in `description`.